test/helpers/chai-plugins.js

```diff
@@ -51,4 +52,69 @@ export default function chaiPlugins(_chai, utils) {
       new chai.Assertion(headers.get(header)).to.equal(expected.toString());
     }
   });
+
+  /**
+   * Determine if a given object has the basic properties of a HAR.
+   *
+   * This does **not** do full HAR validation because we only /really/ actually care about the
+   * contents of the `request` object in `log.entries[*]`. This method is just to give the
+   * `.request`  assertion some nice syntactic prefixing sugar.
+   *
+   * @example await expect(har).to.have.a.har.request;
```

Comment:
Does this need to be awaited? I can't see this doing anything async.

Reply:
The call to verify the HAR is async.